src/components/AboutUsHome/AboutUsHome.jsx

```diff
@@ -0,0 +1,68 @@
+import styles from './AboutUsHome.module.css'
+import menuStyles from '../Header/HamburgerMenu.module.css'
+import chalkyLogo from "../../assets/chalkyLogo.svg"
+import menuButton from '../../assets/menuButton.svg'
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { H1 } from '../../reusableComponents/H1/H1.jsx'
+import { H2 } from '../../reusableComponents/H2/H2.jsx'
+import { P1 } from '../../reusableComponents/P1/P1.jsx'
+import { P2 } from '../../reusableComponents/P2/P2.jsx'
+
```

Comment:
Maybe some comments in the code could be useful to make it easier to navigate the code if you need to change it later on? :)